templates/calendar.html.twig

```diff
@@ -16,9 +16,9 @@
     <div class="row justify-content-center">
         <div class="col-6 ">
             {#test link#}
-            <iframe src="https://calendar.google.com/calendar/embed?src=7vdrj8e80d7sgno2sb48ee1hck%40group.calendar.google.com&ctz=Europe%2FWarsaw" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+            {#<iframe src="https://calendar.google.com/calendar/embed?src=7vdrj8e80d7sgno2sb48ee1hck%40group.calendar.google.com&ctz=Europe%2FWarsaw" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>#}
             {#prod link#}
-            {#<iframe src="https://www.google.com/calendar/embed?showCalendars=0&amp;showTz=0&amp;height=650&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=5jc86vlt0d5d8tqjov0s5h6gho%40group.calendar.google.com&amp;color=%232952A3&amp;ctz=Europe%2FKiev" style=" border-width:0 " width="650" height="650" frameborder="0" scrolling="no"></iframe>#}
+            <iframe src="https://www.google.com/calendar/embed?showCalendars=0&amp;showTz=0&amp;height=650&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=5jc86vlt0d5d8tqjov0s5h6gho%40group.calendar.google.com&amp;color=%232952A3&amp;ctz=Europe%2FKiev" style=" border-width:0 " width="650" height="650" frameborder="0" scrolling="no"></iframe>
```

Comment:
@daria-sieroshtan 
Better put this in the env, so you don't change code at all to switch between test and prod

Reply:
I prefer to keep it simple, this is the only place where it is mentioned